src/Common/src/System/Net/HttpKnownHeaderNames.TryGetHeaderName.cs

```diff
@@ -155,6 +155,7 @@ public static unsafe bool TryGetHeaderName(IntPtr buffer, int length, out string
                         case 'K': potentialHeader = KeepAlive; goto TryMatch;  // [K]eep-Alive
                         case 'S': potentialHeader = SetCookie; goto TryMatch;  // [S]et-Cookie
                         case 'U': potentialHeader = UserAgent; goto TryMatch;  // [U]ser-Agent
+                        case 'R': potentialHeader = RequestId; goto TryMatch;  // [R]equest-Id
```

Comment:
Actually, sorry, I take that back. I think the right fix is to just put that known request header string on a separate type, i.e. have one type for known response header names and one for known request header names.